GVFS/GVFS.Common/Cleanup/GitCleanupStep.cs

```diff
@@ -54,7 +54,7 @@ public void Execute()
         {
             try
             {
-                if (this.RequireCacheLock)
+                if (this.RequireCacheLock && !this.Stopping)

```

Comment:
Is this right? if we are stopping, it looks like we will just go ahead and `CreateProcessAndRun` without the lock?

Reply:
Good call @jamill !  I'll fix this.